packages/application-shell/src/components/navbar/navbar.mod.css

```diff
@@ -381,6 +403,22 @@
   background-color: var(--background-color-for-navbar-when-active);
 }
 
+.item-link:hover .sublist-collapsed-new__active,
+.item-link:hover .sublist-expanded-new__active {
+  display: block;
+  text-align: left;
+  /* stylelint-disable-next-line csstools/value-no-unknown-custom-properties */
+  background-color: var(--background-color-for-navbar-when-active);
+  min-height: 50px;
+  width: 253px;
+  border-radius: 8px;
```

Comment:
Would be nice to not hardcode these values, maybe using design tokens variables. You can also assign some variables in the `:root` object at the top of this file.

Reply:
Yes, true. Thanks! That will be adjusted in the following tasks when styling the submenu and navigation.
But I added the comment just in case. [0a4b394](https://github.com/commercetools/merchant-center-application-kit/pull/3169/commits/0a4b39490b33631f00fe74eacef252806f040bca)